packages/runtime-core/__tests__/components/Suspense.spec.ts

```diff
@@ -537,8 +537,8 @@ describe('Suspense', () => {
     const Comp = {
       setup() {
         const error = ref<Error | null>(null)
-        onErrorCaptured(e => {
-          error.value = e
+        onErrorCaptured(err => {
+          error.value = err instanceof Error ? err : new Error(String(err))
```

Comment:
doesn't https://github.com/vuejs/vue-next/pull/736#issuecomment-587438921 apply also here?

Reply:
Thanks. I fixed it.